web/src/pages/Location/styled.js

```diff
@@ -0,0 +1,45 @@
+import styled, { css } from 'styled-components'
+
+export const Main = styled.main`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
```

Comment:
Cuidado com o _vh_. Acho que não é o caso aqui, mas se a página tivesse scroll, a `Main` só iria ocupar o tamanho da tela (no início do scroll). Quando a intenção for pegar a página toda, o _100%_ vai se sair melhor